static/scss/partials/header.scss

```diff
@@ -194,6 +209,14 @@ header {
     align-items: center;
 }
 
+.mobile-home-link:last-child {
+    display: none;
+
+    @media screen and (max-width: 576px) {
+        display: inline;
```

Comment:
Added this class so that the Sign Up button does not repeat in specific breakpoints (if its in the header, disable it in the dropdown).